zipline/gens/brokers/rh/rh_exceptions.py

```diff
@@ -0,0 +1,34 @@
+"""
```

Comment:
Could you please move the content of this file to robinhood.py? It is not so big, if we remove the comments. The class names are self-explanatory, thus comments are not really needed in my view.